docs/application/tizen-studio/native-tools/leak-sanitizer.md

```diff
@@ -0,0 +1,33 @@
+# Detecting Runtime Memory Leaks with Leak Sanitizer
+
+The Leak Sanitizer is a profiling tool used to detect runtime memory leaks in Tizen native applications. With it, you can discover whether specific parts of code can potentially cause memory leak at runtime. You can also avoid leaks in memory after application execution. The Leak Sanitizer tool detects memory leak at runtime by instrumenting the code during the application compilation. A program with no bugs does not crash when the Leak Sanitizer tool is used, suggesting that the code is safe from potential memory leak.
+
+## Using the Leak Sanitizer
+
+> **Note**
+>
+> Before you run the Leak Sanitizer:
+>
+> - Make sure that you have an emulator or a connected target device running.
+> - If you want to try out the tool and do not have an applicable project to test, create a test project with the Project Wizard using a template or sample. For more information on creating a project, see [Creating Your First Tizen Mobile Native Application](../../native/get-started/mobile/first-app.md).
+
+To use the Leak Sanitizer:
+
+1. In the **Project Explorer** view, right-click the project and select **Profile As > Profile With Leak Sanitizer**.
+
+   The profiling instruments the code and compiles the project for the Leak Sanitizer. Finally, it launches the application on the connected target or emulator.
+
```

Comment:
- Is it profiler ? or profiling?
- What is the intended message that you wish to convey can you please elaborate here. 

The profiling instruments the code and compiles the project for the Leak Sanitizer. Finally, it launches the application on the connected target or emulator.

Reply:
> * Is it profiler ? or profiling?
> * What is the intended message that you wish to convey can you please elaborate here.
> 
> The profiling instruments the code and compiles the project for the Leak Sanitizer. Finally, it launches the application on the connected target or emulator.

Okay, let me fix the description here. 
```
The profiler will link your program against a runtime library containing just the bare necessities required for LeakSanitizer to work by "-fsanitize=leak" option. No compile-time instrumentation will be applied. Finally, it launches the application on the connected target or emulator to detect runtime memory leaks.
```
FYI, origin wiki page is available on https://github.com/google/sanitizers/wiki/AddressSanitizerLeakSanitizer .

Thank you!

Reply:
@dkson95 How about this?  (please check for technical accuracy)
```suggestion
The profiler links your program to a runtime library containing the bare necessities required for LeakSanitizer to work by setting option `-fsanitize` to `leak`,  and no compile-time instrumentation is applied.  Whilst the option is set, the application launches on the connected target or emulator to detect runtime memory leaks.